### PR TITLE
Search part 8 - hash tags

### PR DESCRIPTION
**Pull request by JohnOberhauser**
- Showing hash tags in the hash tags tab
- Adding use cases for following and unfollowing hash tags

### CODE REVIEW

File: core/ui/common/src/main/java/org/mozilla/social/core/ui/common/hashtag/quickview/HashTagQuickView.kt [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    .align(Alignment.Center),
+                painter = MoSoIcons.hash(),
+                contentDescription = "",
+            )
+        }
+
+
+        Spacer(modifier = Modifier.width(MoSoSpacing.sm))
+
+        Column {
```
Comment (devotaaabel) [inline core/ui/common/src/main/java/org/mozilla/social/core/ui/common/hashtag/quickview/HashTagQuickView.kt:56]: Is this column serving a purpose?

**Review by devotaaabel — APPROVED** [review]
Looks good, though I want to reiterate that we really need to spend some time finding ways to reuse paging code using composition over inheritance

File: core/ui/common/src/main/java/org/mozilla/social/core/ui/common/hashtag/quickview/HashTagQuickView.kt (hunk 1 above)
Comment (JohnOberhauser) [reply core/ui/common/src/main/java/org/mozilla/social/core/ui/common/hashtag/quickview/HashTagQuickView.kt]: Nope, I deleted it

**Comment by JohnOberhauser** [comment]
> Looks good, though I want to reiterate that we really need to spend some time finding ways to reuse paging code using composition over inheritance

I'll add something about this in the TIP agenda.

**Pull request merged**